src/DynamoPackages/PackageLoader.cs

```diff
@@ -645,6 +677,16 @@ internal void DoCachedPackageUninstalls(IPreferences preferences)
             {
                 try
                 {
+                    if (pkgNameDirTup.Contains(StandardLibraryDirectory))
+                    {
+                        pkgDirsRemoved.Add(pkgNameDirTup);
+                        Log(
+                             String.Format(
+                                 "Failed to delete standard library package directory at \"{0}\". Uninstalling standard library packages is not allowed.",
+                                 pkgNameDirTup),
+                             WarningLevel.Moderate);
+                        continue;
+                    }
```

Comment:
Do not allow packages from the std lib to be uninstalled by Dynamo. THe alternative is to leave it to folder permissions...